src/model/Model.cpp

```diff
@@ -198,15 +181,13 @@ void Model::generateSQLParts(string &sqlHead, string &sqlBatch, string &sqlUpdat
     
     bool firstField = true;
     
-    for (int i = 0; i < fields.size(); i++)
-    {
+    for (unsigned int i = 0; i < fields.size(); i++) {
         Field* f = fields[i];
         
         bool ignoreField = f->isAutoFilled();
         
         if (!ignoreField) {
-            if (firstField)
-            {
+            if (firstField) {
                 firstField = false;
                 
                 sqlFields +=  f->getName();
```

Comment:
I may be wrong here, but it looks like there is an extra space after the `+=` here.